A32NX/ModelBehaviorDefs/A32NX/A32NX_Interior.xml

```diff
@@ -0,0 +1,55 @@
+<ModelBehaviors>
+
+    <!--
+    Template for defining continuous knobs with lower and upper bounds
+
+    Main Parameters:
+            - ANIM_SIMVAR           <no default>        The name of the simvar to use for the knob
+            - ANIM_SIMVAR_UNITS     Default: "percent"  The simvar units
+            - ANIM_SIMVAR_MIN       Default: 0          The lower bound of the position
+            - ANIM_SIMVAR_MAX       Default: 100        The upper bound of the position
+            - INCREMENT             Default: 1          The increment for each click / hold update
+
+    Authors: Benjamin Dupont (@Benjozork) 10/10/20
+    -->
+    <Template Name="FBW_Continuous_Knob_Finite_From_Simvar">
```

Comment:
Why the FBW prefix? Aren't we using A32NX as a "convention"?

Reply:
So that it's not tied to the A320, future-proofing basically.